### ISSUE TITLE
`ATMForce.getParticleParameters` python wrapper requires displacement vectors

### ISSUE THREAD

**Issue by SimonBoothroyd** [openmm/openmm#4313]
The python wrapper around the `getParticleParameters` method on the new `ATMForce` seems to want you to pass in two `Vec3` pointers that presumably the C++ layer will then update with the displacements set for particle `i`.

Was this the intention, or should the exposed python version of the method take only an index and instead return the `Vec3` displacements?

**Comment by peastman** [comment]
Yes, they should be getting returned as ordinary values.  I'm not sure what's going on.  The SWIG configuration correctly marks them as output arguments:

```
   %apply Vec3 & OUTPUT { Vec3 & displacement1 };
   %apply Vec3 & OUTPUT { Vec3 & displacement0 };
   void getParticleParameters(int index, Vec3 &displacement1, Vec3 &displacement0) const ;
```

But for some reason it's being generated incorrectly.

**Comment by peastman** [comment]
The fix is in #4319.  Once it's merged I'll build a new release candidate.
